helm_deploy/values-test.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# Per environment values which override defaults in approved-premises-ui/values.yaml
+
+generic-service:
+  replicaCount: 2
+
+  ingress:
+    hosts:
+      - temporary-accommodation-test.hmpps.service.justice.gov.uk
+    contextColour: green
+    tlsSecretName: hmpps-temporary-accommodation-test-cert
+
+  env:
+    ENVIRONMENT: test
+    APPROVED_PREMISES_API_URL: 'https://approved-premises-api-test.hmpps.service.justice.gov.uk'
+    INGRESS_URL: 'https://temporary-accommodation-test.hmpps.service.justice.gov.uk'
+    HMPPS_AUTH_URL: 'https://sign-in-test.hmpps.service.justice.gov.uk/auth'
+    TOKEN_VERIFICATION_API_URL: 'https://token-verification-api-test.prison.service.justice.gov.uk'
```

Comment:
I think we should be using `dev` environments for these two

Reply:
Good catch